pkg/app/context.go

```diff
@@ -1059,19 +1059,57 @@ func (ctx *RequestContext) Body() ([]byte, error) {
 type ClientIP func(ctx *RequestContext) string
 
 var defaultClientIP = func(ctx *RequestContext) string {
-	RemoteIPHeaders := []string{"X-Real-IP", "X-Forwarded-For"}
-	for _, headerName := range RemoteIPHeaders {
-		ip := ctx.Request.Header.Get(headerName)
-		if ip != "" {
-			return ip
+	RemoteIPHeaders := []string{"X-Forwarded-For", "X-Real-IP"}
+	TrustedProxies := []string{"0.0.0.0"}
```

Comment:
TrustedProxies should support user configuration rather than overloading entire functions.

Reply:
i got it, thx